exporter/signalfxexporter/exporter.go

```diff
@@ -102,6 +103,17 @@ func newSignalFxExporter(
 				// TODO: What other settings of http.Client to expose via config?
 				//  Or what others change from default values?
 				Timeout: config.Timeout,
+				Transport: &http.Transport{
+					Proxy: http.ProxyFromEnvironment,
+					DialContext: (&net.Dialer{
+						Timeout:   5 * time.Second,
+						KeepAlive: 30 * time.Second,
+					}).DialContext,
+					MaxIdleConns:        int(config.MaxConnections),
```

Comment:
The default transport also has `ForceAttemptHTTP2: true` (both usages).

Reply:
I have added suggested changes. Please review the same.